run-tests.sh

```diff
@@ -21,3 +21,4 @@
 # submit itself to any jurisdiction.
 
 sphinx-build -qnN docs docs/_build/html
+prettier **/*.js --list-different
```

Comment:
should we add a [`.prettifierignore`](https://prettier.io/docs/en/cli.html#ignore-path) for `docs/*`?
Otherwise:
```console
$ prettier '**/*.js' --list-different
docs/_build/html/_static/doctools.js
docs/_build/html/_static/documentation_options.js
docs/_build/html/_static/jquery-3.2.1.js
docs/_build/html/_static/jquery.js
docs/_build/html/_static/searchtools.js
docs/_build/html/_static/underscore-1.3.1.js
docs/_build/html/_static/underscore.js
docs/_build/html/_static/websupport.js
docs/_build/html/searchindex.js
$ echo $?
1
```

Reply:
The command is meant to be executed inside `reana-ui/src`. I will make it more clear :)

Reply:
Solved ✅